kong/runloop/handler.lua

```diff
@@ -509,6 +512,15 @@ do
 
 
   get_plugins_iterator = function()
+    if not plugins_iterator then
+      local ok, err = rebuild_plugins_iterator(PLUGINS_ITERATOR_SYNC_OPTS)
+      if not ok then
+        -- If an error happens while updating, log it and return non-updated
+        -- version
+        log(ERR, "could not rebuild plugins iterator: ", err,
+                 " (stale plugins iterator will be used)")
+      end
```

Comment:
```suggestion
      build_initial_plugins_iterator()
 ```

```lua
local function build_initial_plugins_iterator()
     local ok, err = rebuild_plugins_iterator(PLUGINS_ITERATOR_SYNC_OPTS)
      if not ok then
        -- If an error happens while updating, log it and return non-updated
        -- version
        log(ERR, "could not rebuild plugins iterator: ", err,
                 " (stale plugins iterator will be used)")
      end
end
```

And same with above. Just extract that to function.